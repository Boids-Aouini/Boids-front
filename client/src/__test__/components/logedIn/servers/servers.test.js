import React from 'react';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store';
import { shallow } from 'enzyme';
import Servers from '../../../../components/logedIn/servers/servers';

let mockStore = configureStore();
let initState = {};
let store = mockStore(initState);

describe('<Servers />', () => {
    describe('render()', () => {
        test('render without crashing', () => {
            shallow(<Servers store={store} />);
        })
    })
})