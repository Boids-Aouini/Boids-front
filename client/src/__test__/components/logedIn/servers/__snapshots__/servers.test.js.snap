// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Servers /> render() render component 1`] = `
<Servers
  createServer={[Function]}
  currentChannel={[Function]}
  current_server={[Function]}
  getChannels={[Function]}
  getPosts={[Function]}
  logoutAction={[Function]}
  retreiveServerAsLeader={[Function]}
  retreiveServerAsMember={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
/>
`;
